pandasai/core/generate_chart.py

```diff
@@ -0,0 +1,50 @@
+from chart_generator import ChartGenerator
+from matplotlib_chart_generator import MatplotlibChartGenerator
+from plotly_chart_generator import PlotlyChartGenerator
+from typing import Dict
+
+
+# Chart generator configmap
+chart_generators_map: Dict[str, ChartGenerator] = {
+    "matplotlib": MatplotlibChartGenerator(),
+    "plotly": PlotlyChartGenerator(),
+    "seaborn": None,
+}
+
+def generate_chart(data, library, chart_type, **kwargs):
+    """
+    Generate a data visualization chart based on the provided parameters.
+
+    Args:
+        data (pandas.DataFrame): The data to be visualized.
+        chart_type (str): The type of chart to be generated (e.g., 'bar', 'line', 'scatter', etc.).
+        library (str): The visualization library to be used ('matplotlib', 'seaborn', 'plotly').
+        **kwargs: Additional keyword arguments required for configuring the chosen chart type.
+
+    Returns:
+        The generated chart object.
+    """
+    x=y=None
+    if "x" in kwargs and "y" in kwargs:
+        x=kwargs["x"]
+        y=kwargs["y"]
+        del kwargs["x"]
+        del kwargs["y"]
+    else:
+        print("x and y base parameters should be supplied for the chart generation")
+        return None
```

Comment:
`None` is excessive after return statement, since `None` is return on default when there is no expression left side:
```python
return
```